backend/persistent/datastores/Datastores.js

```diff
@@ -70,13 +71,24 @@ export function query(queryObject, connection) {
 }
 
 /*
- * connect functions attempt to ping the connection and
- * return a promise that is empty
+ * connect attempts to ping the connection and
```

Comment:
Is this JSDocs format?  Should it be @params {object} connection

Reply:
I've added a commit to format it as JSDoc